src/pages/docs/chat/api/javascript/occupancy.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Occupancy
+meta_description: ""
+---
+
+The `Occupancy` interface is used to subscribe to occupancy updates and fetch the occupancy metrics of a room.
+
+Access it via `room.occupancy`.
+
+---
+
+## Get the room occupancy <a id="get"/>
+
+`room.occupancy.get()`
+
+Get the current occupancy of the room.
```

Comment:
"... as an instantaneous value from the server" or something to that effect, implies the REST nature of the call